### PR TITLE
Use #section-name instead of ?section=section-name

### PR DESCRIPTION
**Pull request by seemantk**
@curran ready for your review.

tab-content divs have the appropriate id attributes. Bootstrap doesn't
handle by default (no idea why, but stackoverflow agrees), so we have to
use javascript, but at least it's simple.

### CODE REVIEW

File: js/schematic.js [hunk 1]
```diff
@@ -172,14 +172,14 @@ function getQueryVariables() {
     var vars = {}
```
Comment (curran) [inline js/schematic.js]: @seemantk Looks good! All these variables can also be removed, right?

**Comment by seemantk** [comment]
I have an idea for it still

**Comment by curran** [comment]
Maybe it could store the state of the menus, sort mode, and selected year? We could use the hash or the params for these. Are you thinking of using both the hash *and* the query params?

**Comment by seemantk** [comment]
Right!  Like "#contribution-limits?q=PACTOCAND&y=1998"

**Comment by curran** [comment]
That would be super cool.

**Pull request merged**